Branch-SDK/src/main/java/io/branch/referral/ServerRequest.java

```diff
@@ -559,6 +559,20 @@ private void updateDisableAdNetworkCallouts() {
             }
         }
     }
+
+    private boolean shouldOmitPreinstallData(JSONObject params) {
```

Comment:
nit: fairly minor but could probably be named like `isLinkAttribution` so the function name gives a hint `why` the preinstall data is to be omitted.

Reply:
Perfect